### PR TITLE
Problem: python dependencies in integration tests are old

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- update python to 3.11
- update dependencies

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

## Summary by CodeRabbit

- **Refactor**
	- Updated hashing operations in integration tests for enhanced performance and reliability.
	- Improved code readability and maintenance by aliasing an import used for message creation.

### CODE REVIEW

**Review by mmsqe — APPROVED** [review]

File: integration_tests/poetry.lock [hunk 1]
```diff
@@ -409,201 +624,188 @@ files = [
...
-
-[[package]]
-name = "eth-bloom"
-version = "1.0.4"
-description = "Python implementation of the Ethereum Trie structure"
-optional = false
-python-versions = ">=3.6, <4"
+python-versions = ">=3.8, <4"
 files = [
-    {file = "eth-bloom-1.0.4.tar.gz", hash = "sha256:688317306d87b823da63d24e1ad706defadbd865887ed4bddf7fbd0410b2093c"},
```
Comment (mmsqe) [inline integration_tests/poetry.lock:467]: seem still need eth-bloom

Comment (yihuang) [reply integration_tests/poetry.lock:467]: added

File: integration_tests/test_ibc_rly.py [hunk 2]
```diff
@@ -57,24 +57,56 @@ def ibc(request, tmp_path_factory):
...
+        )
+    ]
+
+
 def coin_received(receiver, amt, denom):
     return {
         "receiver": receiver,
-        "amount": [(amt, denom)],
+        "amount": [
+            AttributeDict(
```
Comment (yihuang) [inline integration_tests/test_ibc_rly.py:75]: You seems overrided the amount_dict fix?

Comment (mmsqe) [reply integration_tests/test_ibc_rly.py]: yes, [amount](https://github.com/cosmos/cosmos-sdk/blob/v0.47.9/x/bank/types/events.go#L36) is coin type and turn to dict in new web3.py

**Pull request merged**